backend/src/models/db/js/tables.js

```diff
@@ -591,38 +360,23 @@ function updateCourseRatings(db, code) {
  * @returns {object} SQLObject
  */
 async function createDB(db) {
-    const timeList = [Date.now()/1000]
-    let promises = []
-    await Promise.all([
-        createUserTable(db),
-        createUniversityTable(db),
-        createSubjectsTable(db),
-        createCourseTable(db),
-        createQuestionTable(db),
-        createReviewTable(db),
-        createCommentTable(db),
-        createLikesTable(db),
-        createDegreesTable(db),
-        createFacultiesTable(db)
-    ])
-        .then(() => {
-            timeList.push(Date.now()/1000)
-            console.log(`Created tables in ${((timeList[1] - timeList[0])).toFixed(3)} seconds`)
-            return Promise.all([initUniTable(db), initSubjectTable(db), initCourseTable(db),
-                initQuestionsTable(db), initReviewTable(db), initFacultiesTable(db), initDegreesTable(db)])
-        })
+    const timeList = [Date.now() / 1000]
+
+    timeList.push(Date.now() / 1000)
+    console.log(`Created tables in ${((timeList[1] - timeList[0])).toFixed(3)} seconds`)
+    Promise.all([initQuestionsTable(db), initReviewTable(db)])
```

Comment:
This chain is broken, investigating now.

Reply:
how so? Also see earlier comment all of this could easily be in same format as init_sql.js to produce a long sql file.